### PR TITLE
WIP [PRO-1851] ostree stuff

### PR DESCRIPTION
**Pull request by Danten**
[WIP] A new package-manager for installing ostree from treehub.

### CODE REVIEW

File: src/package_manager/package_manager.rs [hunk 1]
```diff
@@ -61,6 +64,7 @@ impl PackageManager {
             PackageManager::Off => panic!("no package manager"),
             PackageManager::Deb => "deb".to_string(),
             PackageManager::Rpm => "rpm".to_string(),
+            PackageManager::TreeHub => "thm".to_string(),
```
Comment (stevana) [inline src/package_manager/package_manager.rs]: Does this mean we can deprecate the OsTree stuff?

Comment (Danten) [inline src/package_manager/package_manager.rs]: Maybe, that Ostree stuff was just meant for demo, but I think the deprecation should happen in another commit.

File: src/package_manager/thm.rs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        .env("PULL_URI", pkg.pullUri)
+        .output()
+        .map_err(|e| (UpdateResultCode::GENERAL_ERROR, format!("running script {:?}", e))));
+
+    let stdout = String::from_utf8_lossy(&output.stdout).into_owned();
+    let stderr = String::from_utf8_lossy(&output.stderr).into_owned();
+
+    match output.status.code() {
+        Some(0) => {
+            if (&stdout).contains("already installed") {
```
Comment (taheris) [inline src/package_manager/thm.rs]: Is this output we actually expect to see, or just from the deb package manager (which is also kinda ugly)?

Comment (Danten) [inline src/package_manager/thm.rs]: I copied it from otb.rs which probably copied it from deb yes. I don't actually know what the output is from ostree, which is why I do the same as otb.

**Pull request merged**